src/index.html

```diff
@@ -20,8 +21,476 @@
       href="styles/index.scss"
     />
   </head>
-
+  <!-- endregion -->
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="main_page"
+        class="header__logo"
+      >
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav header__nav">
+          <li>
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <div class="card__content">
+            <div class="card__image">
+              <img
+                src="/src/images/imac.jpeg"
+                width="160px"
+                height="134px"
+                alt="Imac"
+              />
+            </div>
+
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
```

Comment:
This tag used for title of the full page, so you need to use another tag here
```suggestion
            <h2 class="card__title">
              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
            </h2>
```

Reply:
Not fixed